### PR TITLE
virsh_migrate_setmaxdowntime: allow setting downtime any time

### PR DESCRIPTION
**Pull request by PandaWei**
setting max downtime can only be used during migration with
old version of libvirt.
so, if setting max downtime successfully when VM is not during
migration, we can regards it as TestFail.

But it allow setting migration max downtime any time since 1.2.9,
so it's better to check the version before raise TestFail.

### CODE REVIEW

**Comment by PandaWei** [comment]
[root@RX300S6-1 virt]# ./run -t libvirt --no-downloads -k --keep-image-between-tests --tests "type_specific.io-github-autotest-libvirt.virsh.migrate_setmaxdowntime.error_test.do_not_migrate"
SETUP: PASS (0.00 s)
DATA DIR: /var/lib/virt_test
DEBUG LOG: /home/autotest_fjtest_v1.6.1_release/client/tests/virt/logs/run-2015-08-27-02.02.07/debug.log
TESTS: 1
(1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_setmaxdowntime.error_test.do_not_migrate:PASS (114.42 s)
TOTAL TIME: 115.20 s (01:55)
TESTS PASSED: 1
TESTS FAILED: 0
SUCCESS RATE: 100.00 %

File: libvirt/tests/src/virsh_cmd/domain/virsh_migrate_setmaxdowntime.py [hunk 1]
```diff
@@ -160,8 +160,14 @@ def run(test, params, env):
     # Check results.
     if status_error:
         if ret_setmmdt:
-            raise error.TestFail("virsh migrate-setmaxdowntime succeed "
-                                 "but not expected.")
+            if not do_migrate and libvirt_version.version_compare(1, 2, 9):
+                # https://bugzilla.redhat.com/show_bug.cgi?id=1146618
+                # Commit fe808d9 fix it and allow setting migration
+                # max downtime any time since libvirt-1.2.9
+                pass
```
Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/domain/virsh_migrate_setmaxdowntime.py]: Without 'pass' directly, maybe you can add an info log

**Comment by PandaWei** [comment]
Update

**Comment by will-Do** [comment]
Thx
ACK

**Comment by PandaWei** [comment]
Thanks @will-Do 
merge now

**Pull request merged**